### PR TITLE
US004 Task Deleting

### PR DESCRIPTION
**Pull request by kuzivka**
Added possibility to delete tasks

### CODE REVIEW

File: src/components/AddNewTodoModal.tsx [hunk 1]
```diff
@@ -67,7 +67,7 @@ export function AddNewTodoModal(props: IAddNewTodoModalProps) {
             onChange={(newValue) => onDatePicking(newValue)}
```
Comment (Kost927) [inline src/components/AddNewTodoModal.tsx]: make a separate function, do not need add logic in the jsx

File: src/components/TaskCards.tsx [hunk 2]
```diff
@@ -29,6 +43,13 @@ export default function TaskCards() {
             primary={task}
             secondary={`Expires at: ${expiresAt}`}
           />
+          <IconButton
+            aria-label="delete"
+            size="small"
+            onClick={() => deleteOnClick(id)}
```
Comment (Kost927) [inline src/components/TaskCards.tsx]: the same, separate func

Comment (kuzivka) [reply src/components/TaskCards.tsx]: I don't really understand how to make a new func in this case (with passing id parameter)

Comment (kuzivka) [reply src/components/TaskCards.tsx]: done

**Pull request merged**